### ISSUE TITLE
HashMap Data Structure: Set contains only values, not keys

### ISSUE THREAD

**Issue by Prin-S** [TheOdinProject/curriculum#28853]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

There's a sentence in the **Buckets** section that says: `A Set is similar to a hash map but the key difference (pun intended) is that a Set will have nodes with only keys and no values.`

The correct statement should be: `A Set is similar to a hash map but the key difference (pun intended) is that a Set will have nodes with only values and no keys.`

Basically, a set contains only **values**, not keys.

References:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set
- https://javascript.info/map-set

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/javascript-hashmap-data-structure

**Comment by JoshDevHub** [comment]
Closed for reasons stated in #28854
